build/InternalChecksCI-azureBuild-pipeline.yml

```diff
@@ -0,0 +1,172 @@
+name: fhir-ci-$(Date:yyyyMMdd)$(Rev:-r)
+
+parameters:
+- name: BuildConfiguration
+  default: release
+
+variables:
+- name: vmImage
+  value: windows-latest 
+
+trigger:
+
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - /
+    exclude:
+    - /build
+    - /docs
+  batch: True
+
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: main
+
+jobs:
+
+- job: Phase_1
+  displayName: Internal Checks CI - build and restore
+  cancelTimeoutInMinutes: 1
+  pool:
+    name: $(vmImage)
```

Comment:
Since we switched to windows-latest this line needs to be changed to vmImage: $(vmImage). Previously it was referencing an image pool with a specific name instead of a class of VMs by their image type.